npm/vite-dev-server/client/initCypressTests.js

```diff
@@ -24,12 +24,22 @@ if (supportFile) {
 
```

Comment:
Not related to this file but @brian-mann pointed out we will also want to run 

- https://github.com/cypress-io/cypress/blob/develop/system-tests/test/error_ui_spec.ts
- https://github.com/cypress-io/cypress/blob/develop/system-tests/projects/webpack-preprocessor/cypress/e2e/failing.cy.js
- https://github.com/cypress-io/cypress/blob/develop/system-tests/projects/webpack-preprocessor-awesome-typescript-loader/cypress/e2e/failing.cy.ts

As well. These are the ones that verify the typescript compiler is correctly monkey patched to generate the correct source maps. The monkey patch is here: https://github.com/cypress-io/cypress/blob/develop/npm/webpack-preprocessor/lib/typescript-overrides.ts#L39-L45

It's possible this is actually not required. if {webpack,vite}-dev-server already do this. We can find out by running these tests in CT, and see if they pass. 


Ref: https://cypressio.slack.com/archives/D01GR9WNGHF/p1663908419743789

Reply:
I am working this [in another branch](https://github.com/cypress-io/cypress/compare/mikep/21720-ct-stack-traces...mikep/21720-stacktrace-ts-patch) to avoid complicating this PR any further. Will open as a follow-on PR in case there are complications so we don't block the main stacktraces featureset from release - seems to be working as expected once I ported the patch logic over except for a special case for Angular

Reply:
Ok, sounds good